From: Jeroen Roovers <jer@gentoo.org>
Subject: fix typo in parisc support to path_id

As reported at http://bugs.gentoo.org/show_bug.cgi?id=113379 path_id is
still broken for parisc boxes.  This patch from Jeroen Roovers
<jer@gentoo.org> fixes that.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


--- a/extras/path_id
+++ b/extras/path_id
@@ -229,7 +229,7 @@ handle_block_scsi () {
 		;;
 		# PARISC devices
 		*parisc*)
-		adapter="${controler_dev##*/}"
+		adapter="${controller_dev##*/}"
 		bus=parisc;
 		;;
 		*)
